app/lib/task/handlers.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:mime/mime.dart';
+import 'package:pub_dev/shared/exceptions.dart';
+import 'package:pub_dev/shared/handlers.dart';
+import 'package:pub_dev/task/backend.dart';
+import 'package:shelf/shelf.dart' as shelf;
+
+Future<shelf.Response> handleDartDoc(
```

Comment:
Yes, I think so, initially we should probably hide it behind experimental for a while :rofl:

Reply:
Are there other transformations we should apply? Do we have any trusted transformations that **must** be applied.

I'm guessing we will likely have to do something, because these are untrusted files and we're hosting them on a trusted domain...

So it's also possible that we need to find a different subdomain for hosted dartdoc before we ship this.
`api.pub.dev` would be obvious.

Reply:
So right now we do customization before we upload the files, right?

So we should integrate the customization in `pub_worker`, or even better we should do it when we serve.
Because we kind of have to check that the generate HTML is sane...

Reply:
For now, I'll just leave it as is...